src/chart/hawkular-chart-directive.ts

```diff
@@ -446,11 +447,11 @@ module Charts {
           function formatBucketedChartOutput(response) {
             //  The schema is different for bucketed output
             if (response) {
-
               return response.map((point:IChartDataPoint) => {
+                var timestamp = point.start + (point.start - point.end)/2;
```

Comment:
I have some concern with how this will affect some chart types as they will be shifted to the right a bit because of using a midpoint timestamp instead of start timestamp.
This only affects stand alone charts (not normal charts that supply their own data), so its not a big deal.